### ISSUE TITLE
[menu] Some minor imperfections (part 2)

### ISSUE THREAD

**Issue by frosencrantz** [saulpw/visidata#1329]
Noticed several small issues in the vd UI. Great for someone looking to help out and contribute or anyone to do some minor polishing.

- [x] Menu Row > Goto > Next/Previous  are missing Selection '{'-'}'
- [x] Menu Column > missing setcol commands.  To update cells in the column based on the current selection.

Successor to https://github.com/saulpw/visidata/issues/1243  -

**Comment by anjakefala** [comment]
Hi @frosencrantz, these are great suggestions!

I added `go-prev-selected` and `go-next-selected` under `Row` > `Goto` > `next/previous`. You were right that they are missing, and they are commands that I appreciate knowing.

I did not add the `setcol` to Column, because they are actually underneath `Edit` (since they involve modifying data). Maybe that is not the optimal organisational scheme, but I think would merit opening a Discussion instead of an Issue!
